index.js

```diff
@@ -46,7 +46,7 @@ const movieScheme = new Schema({
 
 const Movie = mongoose.model('Movie', movieScheme);
 
-const putAllMoviePagesInDatabase = async () => {
+const populateDB = async () => {
   try {
     let allMovies = [];
     const { results, total_pages } = await getMovies(1);
```

Comment:
@ok2ju is it correct use 'results' inside for? because 'results' is already declared in the upper scope.

Reply:
should i just rename it?